core/src/main/java/com/segment/analytics/kotlin/core/Analytics.kt

```diff
@@ -40,6 +41,12 @@ class Analytics internal constructor(
 
     internal val timeline: Timeline
     val storage: Storage
+    companion object {
+        var debugLogsEnabled: Boolean = false
```

Comment:
whats the use-case of this variable?

Reply:
This is how outside engineers can override the switch from the web.